src/admin-views/aggregator/settings.php

```diff
@@ -475,9 +517,73 @@
 	$ea_disable
 );
 
-$internal = apply_filters( 'tribe_aggregator_fields', $internal );
+$internal = apply_filters( 'tribe_aggregator_fields', $internal, $origin_post_statuses, $origin_categories );
+
+/**
+ * Sort Fields by Priority
+ */
+if ( get_bloginfo( 'version' ) >= 4.7 ) {
+	$internal = wp_list_sort( $internal, 'priority', 'ASC', true );
+}
 
 if ( tribe( 'events-aggregator.main' )->is_service_active() ) {
+
+	$import_setting_links = array(
+		'update-authority'  => array(
+			'name'     => __( 'Update Authority', 'the-events-calendar' ),
+			'priority' => 5,
+		),
+		'global-settings'   => array(
+			'name'     => __( 'Global', 'the-events-calendar' ),
+			'priority' => 10,
+		),
+		'csv-settings'      => array(
+			'name'     => __( 'CSV', 'the-events-calendar' ),
+			'priority' => 15,
+		),
+		'ical-settings'     => array(
+			'name'     => __( 'iCalendar', 'the-events-calendar' ),
+			'priority' => 20,
+		),
+		'ics-settings'      => array(
+			'name'     => __( 'ICS File', 'the-events-calendar' ),
+			'priority' => 25,
+		),
+		'facebook-settings' => array(
+			'name'     => __( 'Facebook', 'the-events-calendar' ),
+			'priority' => 30,
+		),
+		'google-settings'   => array(
+			'name'     => __( 'Google Calendar', 'the-events-calendar' ),
+			'priority' => 35,
+		),
+		'meetup-settings'   => array(
+			'name'     => __( 'Meetup', 'the-events-calendar' ),
+			'priority' => 40,
+		),
+		'url-settings'      => array(
+			'name'     => __( 'Other URLs', 'the-events-calendar' ),
+			'priority' => 45,
+		),
+	);
+
+	/**
+	 * Filter the Import Setting Links on the Import Tab
+	 *
+	 * @since TDB
+	 *
+	 * @param $import_setting_links array an array of import setting anchor links
+	 *
+	 */
+	$import_setting_links = apply_filters( 'tribe_aggregator_setting_links', $import_setting_links );
+
+	/**
+	 * Sort Header List Order
+	 */
+	if ( get_bloginfo( 'version' ) >= 4.7 ) {
+		$import_setting_links = wp_list_sort( $import_setting_links, 'priority', 'ASC', true );
```

Comment:
it is a function that calls a class. Not sure it is worth moving that over both to reorder fields. They would only see Eventbrite at the end and not after csv. Since we only support 4.5 and 4.6 below 4.7 those two versions would have a different order and nothing else. For aggregator to work don't they have  to have 4.7 anyway?

Reply:
yep, changed the branch